### ISSUE TITLE
Pre-work Checklist: Developer: Matthew Chau

### ISSUE THREAD

**Issue by MattChau01** [hackforla/website#4973]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by MattChau01** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took me approximately 4 accumulated hours to assign my good first issue, attending meetings and finishing pre-work items up until this bullet.

**Comment by MattChau01** [comment]
1. Progress: I am currently waiting on my second review on #5112 for issue #5003. Once branch is merged, I will be looking into a small issue to work on next.
2. Blockers: No blockers yet.
3. Availability: I am available for the most part in terms of communication, and will be available on the weekend to work on any additional fixes.
4. ETA: Expected to have reviews finished by end of wekeend.
5. Pictures: Updated meeting date from `Wednesday` to `Thursday`.

![image](https://github.com/hackforla/website/assets/113082803/609ef453-d972-4aeb-b271-ec5a470b754b)

**Comment by MattChau01** [comment]
1. Progress: Currently, I am working on Issue #5009. I left a comment for @roslynwythe on whether actual code would be needed to changed and pushed. Otherwise, I have completed the other steps specified on the issue.
2. Blockers: No blockers on my end, just waiting to hear back on the issue.
3. Availability: I will be available to work on the issue Saturday Morning and Most of Sunday.
4. ETA: I'm looking to have this issue completed by end of day Sunday.

**Comment by MattChau01** [comment]
Progress: I've been assigned to review pull requests: #5263 and #5283.
Blockers: No blockers as of yet.
Availability: I will have time in the evening today, Thursday and most of the weekends.
ETA: I'm expecting to complete these reviews by EoD Friday.

**Comment by MattChau01** [comment]
Progress: The last item I've worked on is a pull request review on #5344. The pull request has already been merged, and will be looking to attend the next standup to pick up another pull request to review.
Blockers: No blockers.
Availability: My availability is usually weekday evenings.

**Comment by MattChau01** [comment]
Progress: I will be working on a pull request review for issue #5558 and #5565 and picked up a small issue #4961.


Blockers: No blockers as of yet.

Availability: I will be available in the evenings this weekend.

ETA: Looking to complete my pull request reviews by tonight, Friday and completing my small issue by end of Sunday.

**Comment by MattChau01** [comment]
Progress: I've successfully closed out my small issue #4961 and will be reviewing a pull request

Blockers: None

Availability: Will be available for reach in the evenings on weekday and most of the day on the weekends

ETA: Looking to complete code review by end of day Friday.

**Comment by MattChau01** [comment]
Progress: Will be working on reviewing pull requests: #5661 and #5664 
Blockers: None
Availability: I can be reached in the evenings
ETA: Looking to complete items by end of Sunday

**Comment by MattChau01** [comment]
Progress: Will be looking through the latest pull requests to do some code reviews
Blockers: None
Availability: Can be reached anytime through the days via Slack

**Comment by MattChau01** [comment]
Progress: Will be working on reviewing pull requests: 5731
Blockers: None
Availability: I can be reached in the evenings
ETA: Looking to complete items by end of today

**Comment by MattChau01** [comment]
Progress: will be reviewing PR 5785
Blockers: none
Availability: evenings
ETA: Looking to complete by end of day Thursday

**Comment by MattChau01** [comment]
Progress: will be reviewing PR 5788
Blockers: none
Availability: evenings
ETA: Looking to complete by end of day Sunday

**Comment by HackforLABot** [comment]
## Activity Log: MattChau01
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- MattChau01 opened: https://github.com/hackforla/website/issues/4973 at 07/17/2023, 06:54 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/4973 at 07/17/2023, 07:31 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/4051#issuecomment-1484017168 at 07/20/2023, 03:06 PM PDT
- MattChau01 unassigned: https://github.com/hackforla/website/issues/4051#issuecomment-1644675439 at 07/20/2023, 03:29 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/4948 at 07/23/2023, 10:01 AM PDT
- MattChau01 opened: https://github.com/hackforla/website/pull/5054 at 07/23/2023, 12:29 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4948#issuecomment-1646941787 at 07/23/2023, 12:34 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5054#issuecomment-1650383266 at 07/25/2023, 12:06 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5054#issuecomment-1652871477 at 07/26/2023, 08:55 PM PDT
- MattChau01 PR was merged: https://github.com/hackforla/website/pull/5054#event-9931236163 at 07/26/2023, 09:45 PM PDT
- MattChau01 issue closed by PR 5054: https://github.com/hackforla/website/issues/4948#event-9931236256 at 07/26/2023, 09:45 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/5003 at 07/27/2023, 01:35 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/5003#issuecomment-1654535514 at 07/27/2023, 01:37 PM PDT
- MattChau01 opened: https://github.com/hackforla/website/pull/5112 at 07/28/2023, 11:00 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/5003#issuecomment-1656569154 at 07/28/2023, 11:04 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1664380607 at 08/03/2023, 10:40 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1664386861 at 08/03/2023, 10:45 AM PDT
- MattChau01 PR was merged: https://github.com/hackforla/website/pull/5112#event-10014600205 at 08/04/2023, 11:44 PM PDT
- MattChau01 issue closed by PR 5112: https://github.com/hackforla/website/issues/5003#event-10014600284 at 08/04/2023, 11:44 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/5009 at 08/08/2023, 10:06 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/5009#issuecomment-1670015075 at 08/08/2023, 10:20 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/5009#issuecomment-1673821223 at 08/10/2023, 12:52 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1675027556 at 08/11/2023, 09:06 AM PDT
- MattChau01 issue closed as completed: https://github.com/hackforla/website/issues/5009#event-10079478735 at 08/13/2023, 02:31 AM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5236#pullrequestreview-1579732478 at 08/15/2023, 08:12 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1690237879 at 08/23/2023, 09:07 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5283#issuecomment-1690379091 at 08/23/2023, 10:44 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5263#issuecomment-1690379274 at 08/23/2023, 10:44 AM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5263#pullrequestreview-1592771584 at 08/23/2023, 09:48 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5283#pullrequestreview-1594925941 at 08/24/2023, 10:56 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5344#issuecomment-1698395120 at 08/29/2023, 07:25 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5344#pullrequestreview-1606241778 at 08/31/2023, 10:20 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1705944660 at 09/04/2023, 09:58 PM PDT
- MattChau01 assigned: https://github.com/hackforla/website/issues/4961 at 09/18/2023, 09:38 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4961#issuecomment-1724821853 at 09/18/2023, 09:41 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5558#issuecomment-1726806267 at 09/19/2023, 07:58 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1732168921 at 09/22/2023, 06:20 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5558#pullrequestreview-1640776429 at 09/22/2023, 08:52 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732206569 at 09/22/2023, 09:23 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5565#pullrequestreview-1641125206 at 09/24/2023, 02:46 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732767012 at 09/24/2023, 06:28 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732888527 at 09/24/2023, 09:28 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732960373 at 09/24/2023, 10:46 PM PDT
- MattChau01 opened: https://github.com/hackforla/website/pull/5586 at 09/25/2023, 11:48 PM PDT
- MattChau01 PR was merged: https://github.com/hackforla/website/pull/5586#event-10481543146 at 09/26/2023, 06:36 PM PDT
- MattChau01 issue closed by PR 5586: https://github.com/hackforla/website/issues/4961#event-10481543243 at 09/26/2023, 06:36 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5565#pullrequestreview-1647888143 at 09/27/2023, 09:57 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1738434526 at 09/27/2023, 10:02 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5616#pullrequestreview-1651647437 at 09/29/2023, 11:30 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1749973776 at 10/05/2023, 09:40 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5664#issuecomment-1749975708 at 10/05/2023, 09:43 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5661#issuecomment-1749980160 at 10/05/2023, 09:50 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5665#issuecomment-1751596327 at 10/06/2023, 09:30 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5664#pullrequestreview-1662997759 at 10/06/2023, 09:58 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5661#pullrequestreview-1663686993 at 10/08/2023, 08:27 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5665#pullrequestreview-1663689302 at 10/08/2023, 08:33 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1760914416 at 10/12/2023, 10:48 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5718#issuecomment-1765137364 at 10/16/2023, 12:26 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5718#pullrequestreview-1683085550 at 10/17/2023, 11:22 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5731#issuecomment-1767550971 at 10/17/2023, 08:08 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1773091872 at 10/20/2023, 10:05 AM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5731#pullrequestreview-1690576715 at 10/20/2023, 10:09 AM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5785#issuecomment-1780401294 at 10/25/2023, 09:36 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1780401996 at 10/25/2023, 09:37 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5785#pullrequestreview-1701086670 at 10/26/2023, 11:38 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/issues/4973#issuecomment-1782381232 at 10/26/2023, 11:40 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5788#pullrequestreview-1703034891 at 10/29/2023, 01:53 PM PDT
- MattChau01 closed as completed: https://github.com/hackforla/website/issues/4973#event-10860132254 at 11/03/2023, 01:06 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5788#issuecomment-1793583259 at 11/04/2023, 04:50 PM PDT
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5828#pullrequestreview-1713890059 at 11/04/2023, 05:07 PM PDT
- MattChau01 commented on: https://github.com/hackforla/website/pull/5893#issuecomment-1807363046 at 11/12/2023, 05:55 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5893#pullrequestreview-1733500030 at 11/15/2023, 08:48 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/5922#issuecomment-1818004090 at 11/19/2023, 02:52 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/5922#pullrequestreview-1743565845 at 11/21/2023, 08:54 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/6003#issuecomment-1853336221 at 12/12/2023, 10:23 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/6003#pullrequestreview-1783004868 at 12/14/2023, 05:39 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/6025#issuecomment-1873622405 at 01/01/2024, 09:02 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/6025#pullrequestreview-1803388403 at 01/03/2024, 07:43 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/6131#issuecomment-1899805999 at 01/18/2024, 09:44 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/6186#issuecomment-1913432540 at 01/27/2024, 07:26 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/6186#pullrequestreview-1847376465 at 01/27/2024, 07:30 PM PST
- MattChau01 commented on: https://github.com/hackforla/website/pull/6270#issuecomment-1935355358 at 02/08/2024, 09:26 PM PST
- MattChau01 submitted review: https://github.com/hackforla/website/pull/6270#pullrequestreview-1873483072 at 02/09/2024, 10:03 PM PST
